starboard/shared/starboard/player/filter/testing/player_components_test.cc

```diff
@@ -382,8 +382,7 @@ class PlayerComponentsTest
   // working.
   void RenderAndProcessPendingJobs() {
     // Call GetCurrentDecodeTarget() periodically for decode to texture mode.
```

Comment:
This comment can be removed or changed since it's not longer limited to just decode to texture mode

Reply:
Fixed in commit 751140a32a7

Reply:
Nit: The commit message of the 2nd commit is the same as the first. The commit message should convey the contents on the commit. In this case it could be e.g. "Fix outdated comment"